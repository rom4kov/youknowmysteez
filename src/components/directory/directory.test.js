import Directory from "./directory.component";
import { shallow } from "enzyme";

it("properly render Directory component", () => {
  expect(shallow(<Directory />)).toMatchSnapshot();
});
