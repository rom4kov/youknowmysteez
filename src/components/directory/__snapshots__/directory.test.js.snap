// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`properly render Directory component 1`] = `
<styled.div>
  <DirectoryItem
    category={
      Object {
        "id": 1,
        "imageUrl": "shirts.jpg",
        "route": "shop/hats",
        "title": "TEES",
      }
    }
    key="1"
  />
  <DirectoryItem
    category={
      Object {
        "id": 2,
        "imageUrl": "jackets.jpg",
        "route": "shop/jackets",
        "title": "JACKETS",
      }
    }
    key="2"
  />
  <DirectoryItem
    category={
      Object {
        "id": 3,
        "imageUrl": "sneakers.jpg",
        "route": "shop/sneakers",
        "title": "SNEAKERS",
      }
    }
    key="3"
  />
  <DirectoryItem
    category={
      Object {
        "id": 4,
        "imageUrl": "womens.jpg",
        "route": "shop/womens",
        "title": "WOMENS",
      }
    }
    key="4"
  />
  <DirectoryItem
    category={
      Object {
        "id": 5,
        "imageUrl": "mens.jpg",
        "route": "shop/mens",
        "title": "MENS",
      }
    }
    key="5"
  />
</styled.div>
`;
